cloudify_cli/bootstrap/bootstrap.py

```diff
@@ -322,3 +330,45 @@ def _copy_agent_key(agent_local_key_path, agent_remote_key_path,
     with fabric.settings(**fabric_env):
         fabric.put(agent_local_key_path, agent_remote_key_path, use_sudo=True)
     return agent_remote_key_path
+
+
+def _upload_resources(manager_node, fabric_env, management_ip, rest_port,
+                      protocol):
+
+    upload_resources = \
+        manager_node.properties['cloudify'].get('upload_resources')
+
+    # upload the plugin using CLI upload command
+    rest_client = \
```

Comment:
it is not really expensive the get the rest client so there is not real need to check if `plugin_resources` exists or not, simply create it and it will be used if needed